tests/get-real-file-path-test.js

```diff
@@ -0,0 +1,11 @@
+'use strict';
+
+var assert = require('assert');
+var getRealFilePath = require('../lib/get-real-file-path');
+
+describe.skip('getRealFilePath', function() {
```

Comment:
@dcombslinkedin can you implement these tests

Reply:
Done, inside resolve-package-path-test.js. Checking in shortly.